### ISSUE TITLE
Timeline

### ISSUE THREAD

**Issue by Mucki526** [jlawyerorg/j-lawyer-org#342]
Ich hätte heute die Timeline gebraucht, um festzustellen, wann ein Mandant angerufen hat (habe die Akte am Bildschirm elektronisch geöffnet). Das Öffnen der Akte erscheint aber wohl nicht in der Timeline??

**Comment by j-dimension** [comment]
Protokolliert werden alle Aktionen welche Daten **verändern**.

Wenn auf Basis des Telefonates etwas getan wurde (bspw Notiz oder Dokument erstellt) dann sollte das auch in der Historie stehen.

Ist das ein häufiger Bedarf?

**Comment by Mucki526** [comment]
Ja, eigentlich schon, wenn man die Timeline intensiver nutzen will. Bei Abrechnung nach Zeit oder Effizienz der eigenen Arbeit - ich öffne die Akte und schließe die Akte oder schließe die letzte bearbeitete Datei. Oder ich bekomme ein Telefonat und öffne die Akte für eine Notiz (ich kann natürlich auch hinterher die Länge des Telefonats aus der Fritzbox nehmen. Oder ich suche eine Telefonnummer vom Mandanten der irgendwann am Vormittag angerufen hat und könnte die aus der Liste einfach schnell picken, wenn ich weiss, wann ich seine Akte geöffnet habe. (Mandant hatte neue Telefonnummer - in Akte falsch und ich musste ihn dringend zurückrufen). Der Gründe sind also mehrere. Der Bedarf kommt mit der Möglichkeit und umgekehrt. Es wäre für mich auch logisch - Akte geöffnet - Akte geschlossen in der Timline zu haben (oder optional zu haben). Evt. in anderer Farbe als das bearbeiten der Akte....
